gf-webapp/src/logic/contactsPage/ContactsPage.tsx

```diff
@@ -1,45 +1,29 @@
 import React, {useRef, useState} from "react";
 import {useDictionary} from "../DictionaryContext/useDictionary";
-import {send} from "emailjs-com";
 import {PageBorder} from "../../component/pageBorder/PageBorder";
+import {MailApiService} from "src/service/MailApi/MailApi";
+import {ApiAskAdminPostRequest} from "src/autogenerated/gf-server-api/models/ApiAskAdminPostRequest";
 import styles from "./ContactsPage.module.scss";
 import {Button} from "gf-ui-lib/components/Button/Button";
 
+const defaultAskAdminFormData: ApiAskAdminPostRequest = {name: "", email: "", phone: "", message: ""};
 
 export const ContactsPage = () => {
   const {contactsPage, askAdminForm} = useDictionary().dictionary;
+  const [askAdminFormData, setAskAdminFormData] = useState<ApiAskAdminPostRequest>(defaultAskAdminFormData);
 
-  const [toSend, setToSend] = useState({
-    to_name: "",
-    message: "",
-    phone: "",
-    reply_to: "",
-    check_in: "",
-    check_out: "",
-    room_number: "",
-    adults_number: "",
-    children_number: "",
-  });
+  const sendMail = async () => {
+    const res = await MailApiService.askAdminMessage({apiAskAdminPostRequest: askAdminFormData});
+  };
 
-  const onSubmit = (e: React.ChangeEvent<HTMLFormElement>) => {
+  const onSubmit = async (e: React.ChangeEvent<HTMLFormElement>) => {
     e.preventDefault();
-    send(
-      "service_q2x5wyb",
-      "template_iq8l9cq",
-      toSend,
-      "Iz-maWBTKYHMeQmxB",
-    )
-      .then((response) => {
-        console.log("SUCCESS!", response.status, response.text);
-      })
-      .catch((err) => {
-        console.log("FAILED...", err);
-      });
-    e.target.reset();
+    setAskAdminFormData(defaultAskAdminFormData);
```

Comment:
why do we need set default data on submit?

Reply:
Deleted set default values in inputs